content/docs/introduction/billing.md

```diff
@@ -0,0 +1,188 @@
+
+To cancel your subscription to a Neon paid plan:
+
+1. Navigate to the **Billing** page in the Neon Console.
+1. Click **Cancel subscription**.
+1. Enter your cancellation request and click **Submit**.
+
+This action initiates the cancellation. If your data exceeds [Free Tier](/docs/introduction/technical-preview-free-tier) storage limits, you will be contacted by the Neon Support team with a request to reduce your storage before the paid plan is canceled and Free Tier limits are applied.
+
+## Billing metrics explained
+
+This section provides a detailed explanation of Neon's billing metrics and how they are calculated. Billing in Neon is account-based. For the billing rate for each metric, see [Billing rates](#billing-rates).
+
+<Admonition type="note">
+The **Project storage**, **Written data**, and **Data transfer** billing metrics are calculated in gibibytes (GiB), otherwise known as binary gigabytes. One gibibyte equals 2<sup>30</sup> or 1,073,741,824 bytes.
+</Admonition>
+
+### Compute time
+
+The _Compute time_ metric counts _Compute Unit (CU)_ active time. A CU has 1 vCPU and 4 GB of RAM. In Neon, a [compute endpoint](/docs/reference/glossary/#compute-endpoint) can have anywhere from .25 CUs to 7 CUs. The number of CUs determines the processing capacity of your Neon database. A connection from a client or application activates a compute endpoint and its CUs. Activity on the connection keeps the compute endpoint and its CUs in an active state. A defined period of inactivity places the compute endpoint and its CUs into an idle state.
+
+Factors that affect the amount of compute time include:
+
+- The number of active compute endpoints
+- The number of CUs per compute endpoint
+- Neon's _Auto-suspend compute_ feature, which suspends a compute endpoint (and its CUs) after a period of inactivity. The current default is five minutes.
+- Neon's _Configurable auto-suspend compute_ feature, which allows you to configure the timeout period for the  _Auto-suspend compute_ feature (_coming soon_).
+- Neon's _Always-on_ compute feature, which keeps an endpoint active indefinitely to avoid the few seconds of connection latency when waking an idle compute (_coming soon_).
+- Neon's _autoscaling_ feature, which allows you to set a minimum and maximum number of CUs for each compute endpoint. The number of active CUs scale up and down based on workload (_coming soon_).
+
+
+The cost calculation for _Compute time_ is as follows:
+
+```text
+compute units * active time (hours) * price per hour
+```
+
+### Project storage
+
+The _Project storage_ metric counts the amount of data stored in all of your Neon projects. Project storage includes:
+
+- **Logical data size**
+
+  The size of all databases in your Neon projects. You can think of this as a _snapshot_ of your data at a point in time.
+
+- **Retained Write-Ahead Log (WAL) records**
+
+  Neon retains Write-Ahead Log (WAL) records to support _point-in-time restore_ and _database branching_.
```

Comment:
```suggestion
  Neon retains history to support _point-in-time restore_ and _database branching_.
```
@hlinnaka , what is the right way to mention the WAL when explaining the history size? could you suggest a wording?

Reply:
Revised to focus on "history" as the second component of Project storage size